### ISSUE TITLE
Join the Gitter chat room

### ISSUE THREAD

**Issue by Bluscream** [AdguardTeam/AdguardKnowledgeBase#4]
https://gitter.im/AdguardHQ/AdguardKnowledgebase

**Comment by vozersky** [comment]
@Bluscream hi! 
What is it good for? :)

**Comment by Bluscream** [comment]
It's good if you want a conversation with a customer with quicker response times (like when you're remote controlling his pc)
Also 'ts nice to have a place where you don't need to create a new thread/issue for the smallest things.

**Comment by vozersky** [comment]
I am afraid we don't really have enough human resource for live chat atm. I suggest using reddit/forum. But it's a good idea, we should keep it in mind.

**Comment by ameshkov** [comment]
In fact I find this idea pretty good, instead of using slack for discussing filters-related issues we could have a public channel for that.

**Comment by vozersky** [comment]
true, this can be a good thing for filters or any other more in-depth discussion

**Comment by Bluscream** [comment]
@ameshkov any news on a official gitter from your side?

**Comment by ameshkov** [comment]
@Bluscream not until I am back from vacation:)

**Comment by Bluscream** [comment]
Oh okay enjoy it and don't peek on here to often ^~^

**Comment by vozersky** [comment]
We've [decided](https://forum.adguard.com/index.php?threads/adguard-channel-in-telegram-a-place-for-discussion.22160/) to go for Telegram. Here's the[ channel.](https://t.me/adguard_en)

**Comment by Bluscream** [comment]
Thanks for the invite :)
